examples/simulation/Tutorial2/macros/compare_seed_value.C

```diff
@@ -0,0 +1,20 @@
+int compare_seed_value(TString filename, UInt_t initial_seed) {
+
+  // Open parameter file and get the stored random seed from the
+  // FairBaseParSet parameter container
+  // Compare the random seed with the expected one set during the
+  // transport run
+
+  std::unique_ptr<TFile> myFile( TFile::Open(filename) );
+  std::unique_ptr<FairBaseParSet> par(myFile->Get<FairBaseParSet>("FairBaseParSet"));
```

Comment:
Are you sure `Get` returns ownership?

Reply:
Looking at the [source of `TDirectoryFile::Get`](https://root.cern.ch/doc/master/TDirectoryFile_8cxx_source.html#l00937) (and especially the first `idcur` part) it looks like this returns a non-owning pointer.

Reply:
I don't know. Why do you ask?

Reply:
Because if you store it in a `unique_ptr` as you do, it will `delete` it once it goes out of scope. So, your code makes two assumptions:
1. `Get` passes ownership to you
2. The pointer points to "`delete`able" heap memory

Reply:
No I get your point. I think both assumptions are correct in the example.